trivy.yaml

```diff
@@ -0,0 +1,3 @@
+timeout: 20m
+scan:
+  offline-scan: true
```

Comment:
Can we put these two files somewhere other than the repo root, to avoid cluttering up the root dir? Looks like we could relatively easily move `trivy.yaml` as that's explicitly specified, and it looks like you can specify `--ignorefile value` to move `.trivyignore`.

We can either move them alongside `scanning.yml`, or add a new "scripts" or "config" dir or something? (But if the latter, then we need to bikeshed the name :-)

Reply:
Sounds good! I placed `trivy.yaml` and `.trivyignore` under a new directory - `.github/files/`, reason being I wanted it to be under .github as it relates to CI/CD and I don't want to have any unexpected behavior with .yaml files under .github/workflows - I don't know whether Github would try to parse `trivy.yaml` as a workflow and complain on the syntax. That's just a suggestion, if you're fine with the approach of having that under .github, the new folder name can be discussed as "files" might be too broad. I thought about naming it "x-files" as it does contain an X amount of files and is a funny reference, though I abstained.

Reply:
Yeh using the `.github/` directory is nicer for sure. In other projects, I've just used `.github/foobar.yaml` without the extra subdirectory and not had any issues fwiw.

Reply:
Just using `.github/` per @jnsgruk sounds good. Though if we're going with a subdirectory I'd use "config" as everything is "files".

Reply:
I'm fine with using `.github/`, I hadn't considered that and it does sound good. Agree that "files" is way too generic.